### DISCUSSION TITLE
Fluent-Refresh 1.6 beta 2

### DISCUSSION THREAD

**Discussion by bonzibudd** [bonzibudd/Fluent-Refresh#2] [Announcements]
### Requires that Segoe UI Variable font is installed!
**Proceed on Win11 at your own risk. No guarantees in terms of functionality.**

This is a testing release of Fluent-Metro 1.6. This beta includes these changes:

 - Added new split selector for sub-menu
 - Updated non-curved sub-menu selectors
 - Adjusted padding on compact sub-menu
 - Improved scaling of Shutdown button height
 - Updated size of programs list
 - Updated search header selection

The tiled layout can still be enabled from the XML file of the latest Fluent-Metro release.

Feel free to create a post on the discussions page if you have any questions or concerns, however I may or may not address them at this point in time. Please refer to the Fluent-Metro repository for information and support on the latest stable version.

<hr /><em>This discussion was created from the release <a href='https://github.com/bonzibudd/Fluent-Refresh/releases/tag/v1.6-beta.2'>Fluent-Refresh 1.6 beta 2</a>.</em>

**Comment by peperonEZZ** [comment]
trasnparency doesn't work
